hierarchicalforecast/methods.py

```diff
@@ -481,22 +507,29 @@ class MinTrace:
     
     **Parameters:**<br>
     `method`: str, one of `ols`, `wls_struct`, `wls_var`, `mint_shrink`, `mint_cov`.<br>
+    `nonnegative`: bool, reconciled forecasts should be nonnegative?<br>
 
     **References:**<br>
     - [Wickramasuriya, S. L., Athanasopoulos, G., & Hyndman, R. J. (2019). \"Optimal forecast reconciliation for
     hierarchical and grouped time series through trace minimization\". Journal of the American Statistical Association, 
     114 , 804–819. doi:10.1080/01621459.2018.1448825.](https://robjhyndman.com/publications/mint/).
+    - [Wickramasuriya, S.L., Turlach, B.A. & Hyndman, R.J. (2020). \"Optimal non-negative
```

Comment:
Is this character `\"` needed? I think simple quotation marks suffice.

Reply:
I don't know. I just followed your convention. See the previous reference. Would you like to change each reference?

Reply:
At some point I had some issues with Microsoft parsing the curly quotation marks, I might have overcompensated.
I am unsure now if simple quotations compile correctly in nbdev. Would be good to check again.